spec/models/claim/base_claim_spec.rb

```diff
@@ -523,6 +523,39 @@ class MockSteppableClaim < Claim::BaseClaim
       specify { is_expected.to be_nil }
     end
   end
+
+  describe '#unread_messages_for' do
+    let(:claim) { create(:submitted_claim) }
+    let(:user) { claim.external_user.user }
+
+    it 'returns an empty array if there are no messages' do
```

Comment:
I also think, mentioned in [betterspec.org](https://www.betterspecs.org/) we should make use of contexts and have setup for tests in a before block.

example
```
context 'when there are no messages for user' do
  it { is_expected.to eq([]) }
  # alternative composite spec
  it { is_expected.to be_an(Array).and be_empty }
end
```